srcmedia/ts/membership-graphs.ts

```diff
@@ -0,0 +1,236 @@
+import * as d3 from "d3"
+
+
+// membership data - defined in the global scope outside this module, in the template
+type MembershipCount = {
+    count: number,
+    startDate: string,
+}
+type MembershipCountArray = Array<MembershipCount>
+
+type MembershipTotals = {
+    cards: MembershipCountArray,
```

Comment:
could these just be `cards: Array<MembershipCount>`, or does `MembershipCountArray` get used for something else?

Reply:
that seems obvious now that you point it out